src/main/java/algorithms/curated170/easy/CountingElementsSet.java

```diff
@@ -0,0 +1,52 @@
+package algorithms.curated170.easy;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class CountingElementsSet {
+    public int countElements(int[] arr) {
+
+        Set<Integer> nums = new HashSet<>();
+        int count = 0;
+        for (int n : arr) {
+            nums.add(n);
+        }
+        for (int n : arr) {
+            if (nums.contains(n + 1)) {
+                count++;
+            }
+        }
+        return count;
+    }
```

Comment:
This is quite fine as well.
![image](https://user-images.githubusercontent.com/60903744/118373547-2f7cfa00-b5c0-11eb-938b-7481f011d0f0.png)

Reply:
How does the current solution with sorting perform?

Reply:
![image](https://user-images.githubusercontent.com/60903744/118373732-f729eb80-b5c0-11eb-8c00-bfc94ef08586.png)